packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.ts

```diff
@@ -6,36 +6,71 @@ export const languageError = 'The list of languages cannot be empty';
 
 const languageEmptyError = 'The language must be a non-empty string';
 
-const defaultQuery = /* GraphQL */ `
-  query SitemapQuery(
-    $siteName: String!
-    $language: String!
-    $includedPaths: [String]
-    $excludedPaths: [String]
-    $pageSize: Int = 10
-    $after: String
-  ) {
-    site {
-      siteInfo(site: $siteName) {
-        routes(
-          language: $language
-          includedPaths: $includedPaths
-          excludedPaths: $excludedPaths
-          first: $pageSize
-          after: $after
-        ) {
-          total
-          pageInfo {
-            endCursor
-            hasNext
-          }
-          results {
-            path: routePath
+const defaultQuery = /* GraphQL site path query without personalization (works on sxp and xm cloud) */ `
```

Comment:
This comment needs to be exactly `/* GraphQL */` for syntax highlighting on the query to work in VS Code.